hedera-node/configuration/mainnet/application.properties

```diff
@@ -1 +1,10 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
```

Comment:
We can delete this property, it doesn't exist anymore.

Reply:
Done. Thanks @tinker-michaelj